packages/features/ee/workflows/api/scheduleEmailReminders.ts

```diff
@@ -43,24 +38,6 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
     return;
   }
 
-  // delete batch_ids with already past scheduled date from scheduled_sends
-  const remindersToDelete: { referenceId: string | null }[] = await getAllRemindersToDelete();
-
-  const deletePromises: Promise<any>[] = [];
-
-  for (const reminder of remindersToDelete) {
-    const deletePromise = deleteScheduledSend(reminder.referenceId);
-    deletePromises.push(deletePromise);
-  }
-
-  Promise.allSettled(deletePromises).then((results) => {
-    results.forEach((result) => {
-      if (result.status === "rejected") {
-        logger.error(`Error deleting batch id from scheduled_sends: ${result.reason}`);
-      }
-    });
-  });
```

Comment:
> When a Batch is canceled, all messages associated with that batch will stay in your sending queue, but when their send_at value is reached, they will be discarded instead of attempting delivery.

Interesting, I thought we used to have an issue that they weren't deleted from the sending queue automatically. But also, this shouldn't effect workflow reminders because we would only delete them when they are already in the past

Reply:
https://github.com/calcom/cal.com/blob/main/packages/features/ee/workflows/lib/getWorkflowReminders.ts#L87 . lte also gets the current and past cancelled workflows

Reply:
is the issue that we use `lte` instead of `lt`? Deleting the ones in the past shouldn't cause any issues, right?

Reply:
Yeah `lt` should fix this as we will be only clearing the past events and not the current events.

Reply:
> Yeah lt should fix this as we will be only clearing the past events and not the current events.

Can we do that then instead? 

I was just trying to test it locally if a batch id gets deleted from` /scheduled_sends ` when the email is in the past. But for me that didn't happen and it seems like we do have to manually delete them when they are in the past

Reply:
Yep ! done that !